src/Shared/ProblemDetails/HttpValidationProblemDetailsJsonConverter.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Text.Json;
+using System.Text.Json.Nodes;
```

Comment:
What's this for?

Reply:
Nothing. I have no idea how I did not get build errors because of that.